### ISSUE TITLE
Unable to build to exe

### ISSUE THREAD

**Issue by leamlidara** [conduit-dart/conduit#98]
**Describe the bug**
I cannot compile the source to exe using (dart compile exe SOURCE.dart)

**To Reproduce**
1. dart create project
2. Copy 'conduit: ^3.1.2' to pubspec.yaml
3. Copy some code from document + add some route to resource controllers
4. dart compile exe SOURCE.dart

**Error**
```
error: import of dart:mirrors is not supported in the current Dart runtime

Error: AOT compilation failed
Generating AOT snapshot failed!
```

**Desktop:**
 - OS: Windows 11 Pro
 - Dart SDK version: 2.17.5 (stable) (Tue Jun 21 11:05:10 2022 +0200) on "windows_x64"

**Comment by mitryp** [comment]
I was going to compile my web app too and noticed that issue.
It seems that Dart bundled compiler cannot compile the source correctly.

You can use the `conduit build` command to build the app.
[Gitbook](https://gitbook.theconduit.dev/deploy/build)

**Comment by j4qfrost** [comment]
should be fixed with 3.2.5
